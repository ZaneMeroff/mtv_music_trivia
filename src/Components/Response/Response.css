.response-contianer {
  align-items: center;
  display: flex;
  flex-direction: column;
  height: 70%;
  justify-content: center;
  text-align: center;
}

.response-text {
  color: #fff200;
  font-size: 300%;
  transform: rotate(-20deg);
}

@media only screen and (max-width: 490px) {

  .response-text {
    font-size: 200%;
  }

}
