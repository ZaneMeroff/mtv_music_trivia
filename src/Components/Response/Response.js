import React from 'react';
import './Response.css';
import PropTypes from 'prop-types';

const Response = props => {
    return (
      <div className='response-contianer'>
        <p className='response-text'>{props.text}</p>
      </div>
    )
}

Response.propTypes = {
  text: PropTypes.string,
}

export default Response;
