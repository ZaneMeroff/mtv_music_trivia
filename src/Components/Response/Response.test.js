import React from 'react';
import Response from './Response';
import { shallow } from 'enzyme';

describe('Loading', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Response text='RIGHT!' />)
  })

  it ('should render a Response component', () => {
    expect(wrapper).toMatchSnapshot()
  })

})
