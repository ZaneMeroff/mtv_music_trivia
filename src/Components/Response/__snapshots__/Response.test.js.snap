// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should render a Response component 1`] = `
<div
  className="response-contianer"
>
  <p
    className="response-text"
  >
    RIGHT!
  </p>
</div>
`;
